### PR TITLE
Translate DOM Elements

### PR DESCRIPTION
**Pull request by LuisRevillaM**
Translating **DOM Elements** section of the **API Reference** to Spanish.
Relates to issue #25

### CODE REVIEW

**Comment by LuisRevillaM** [comment]
I think I don't have permissions to assign you the PR, so I'm mentioning you here @dmoralesm , @carburo, @alejandronanez. Thank you!

File: content/docs/reference-dom-elements.md [hunk 1]
```diff
@@ -48,23 +50,23 @@ function MyComponent() {
...
+>Algunos ejemplos en la documentación usan el atributo `style` por conveniencia, pero **generalmente no se recomienda usar el atributo `style` como medio principal para estilizar elementos**. En la mayoría de los casos, [`className`](#classname) debe ser usado para hacer referencia a clases definidas en documentos CSS externos. En React, el atributo `style` se usa con mayor frecuencia para añadir estilos computados dinámicamente al momento de renderización. Revisa tambièn [FAQ: Styling and CSS](/docs/faq-styling.html).
 
-The `style` attribute accepts a JavaScript object with camelCased properties rather than a CSS string. This is consistent with the DOM `style` JavaScript property, is more efficient, and prevents XSS security holes. For example:
+El atributo `style` acepta un objeto de Javascript con propiedades escritas en formato camelCase, en lugar de un CSS string. Esto es consistente con la propiedad DOM `style` en Javascript, es más eficiente y previene vulnerabilidades XSS. Por ejemplo:
```
Comment (carburo) [inline content/docs/reference-dom-elements.md]:
```suggestion
El atributo `style` acepta un objeto de Javascript con propiedades escritas en formato *camelCase*, en lugar de un *string* CSS. Esto es consistente con la propiedad DOM `style` en Javascript, es más eficiente y previene vulnerabilidades XSS. Por ejemplo:
```

File: content/docs/reference-dom-elements.md [hunk 2]
```diff
@@ -77,66 +79,64 @@ function HelloWorldComponent() {
...
 }
 ```
+Las propiedades del objeto aceptado por `style` tienen formato camelCase para ser consistentes con la forma en que se accede a los estilos de los nodos DOM en JS (p.ej `node.style.backgroundImage`). Los prefijos de los proveedores de navegadores, [a excepción de `ms`](http://www.andismith.com/blog/2012/02/modernizr-prefixed/), deben iniciarse con letra mayúscula. Por esto `WebkitTransition` tiene una "W" mayúscula. 
 
-Style keys are camelCased in order to be consistent with accessing the properties on DOM nodes from JS (e.g. `node.style.backgroundImage`). Vendor prefixes [other than `ms`](http://www.andismith.com/blog/2012/02/modernizr-prefixed/) should begin with a capital letter. This is why `WebkitTransition` has an uppercase "W".
-
-React will automatically append a "px" suffix to certain numeric inline style properties. If you want to use units other than "px", specify the value as a string with the desired unit. For example:
+React adjuntará automáticamente el sufijo "px" a ciertas propiedades numéricas. Si quieres usar unidades diferentes a "px", especifica el valor como un string con la unidad deseada. Por ejemplo:
```
Comment (carburo) [inline content/docs/reference-dom-elements.md]:
```suggestion
React adjuntará automáticamente el sufijo "px" a ciertas propiedades numéricas. Si quieres usar unidades diferentes a "px", especifica el valor como un *string* con la unidad deseada. Por ejemplo: 
```

File: content/docs/reference-dom-elements.md [hunk 3]
```diff
@@ -77,66 +79,64 @@ function HelloWorldComponent() {
...
+Si usas renderización de React del lado del servidor, normalmente hay una advertencia cuando el servidor y el cliente presentan contenido diferente. Sin embargo, en algunos casos raros, es muy difícil o imposible garantizar una coincidencia exacta. Por ejemplo, se espera que las marcas de tiempo difieran en el servidor y en el cliente. 
 
-If you set `suppressHydrationWarning` to `true`, React will not warn you about mismatches in the attributes and the content of that element. It only works one level deep, and is intended to be used as an escape hatch. Don't overuse it. You can read more about hydration in the [`ReactDOM.hydrate()` documentation](/docs/react-dom.html#hydrate).
+Si estableces `suppressHydrationWarning` como `true`, React no te advertirá sobre los desajustes en los atributos y el contenido de ese elemento. Solo funciona a un nivel de profundidad y está diseñado para ser utilizado como escotilla de escape. No lo uses en exceso. Puedes leer más sobre hidratación en la [documentación de `ReactDOM.hydrate()`](/docs/react-dom.html#hydrate).
```
Comment (carburo) [inline content/docs/reference-dom-elements.md]:
```suggestion
Si estableces `suppressHydrationWarning` como `true`, React no te advertirá sobre los desajustes en los atributos y el contenido de ese elemento. Solo funciona a un nivel de profundidad y está diseñado para ser utilizado como una vía de escape. No lo uses en exceso. Puedes leer más sobre hidratación en la [documentación de `ReactDOM.hydrate()`](/docs/react-dom.html#hydrate).
```

File: content/docs/reference-dom-elements.md [hunk 4]
```diff
@@ -77,66 +79,64 @@ function HelloWorldComponent() {
...
 
 ### value
 
-The `value` attribute is supported by `<input>` and `<textarea>` components. You can use it to set the value of the component. This is useful for building controlled components. `defaultValue` is the uncontrolled equivalent, which sets the value of the component when it is first mounted.
+El atributo `value` es compatible con los componentes` <input> `y` <textarea> `. Puedes usarlo para establecer el valor del componente. Esto es útil para construir componentes controlados. `defaultValue` es el equivalente no controlado, que establece el valor del componente cuando se monta por primera vez.
 
-## All Supported HTML Attributes
+## Todos los atributos HTML soportados
```
Comment (carburo) [inline content/docs/reference-dom-elements.md]:
```suggestion
## Todos los atributos HTML admitidos
```

File: content/docs/reference-dom-elements.md [hunk 5]
```diff
@@ -77,66 +79,64 @@ function HelloWorldComponent() {
...
 
-As of React 16, any standard [or custom](/blog/2017/09/08/dom-attributes-in-react-16.html) DOM attributes are fully supported.
+A partir de React 16, cualquier atributo de DOM estándar [o personalizado](/blog/2017/09/08/dom-attributes-in-react-16.html) es totalmente soportado.
```
Comment (carburo) [inline content/docs/reference-dom-elements.md]:
```suggestion
A partir de React 16, se admite cualquier atributo de DOM estándar [o personalizado](/blog/2017/09/08/dom-attributes-in-react-16.html).
```

File: content/docs/reference-dom-elements.md [hunk 6]
```diff
@@ -154,7 +154,8 @@ sizes span spellCheck src srcDoc srcLang srcSet start step style summary
 tabIndex target title type useMap value width wmode wrap
 ```
 
-Similarly, all SVG attributes are fully supported:
+Similarmente, todos los atributos SVG son totalmente compatibles con React:
```
Comment (carburo) [inline content/docs/reference-dom-elements.md]:
```suggestion
Similarmente, se admiten todos los atributos SVG: 
```

File: content/docs/reference-dom-elements.md [hunk 7]
```diff
@@ -193,4 +194,4 @@ xlinkHref xlinkRole xlinkShow xlinkTitle xlinkType xmlns xmlnsXlink xmlBase
 xmlLang xmlSpace y y1 y2 yChannelSelector z zoomAndPan
 ```
 
-You may also use custom attributes as long as they're fully lowercase.
+También puedes usar atributos personalizados siempre que estén en minúsculas.
```
Comment (carburo) [inline content/docs/reference-dom-elements.md]:
```suggestion
También puedes usar atributos personalizados siempre que estén completamente en minúsculas.
```

**Comment by LuisRevillaM** [comment]
> Thank you @LuisRevillaM for your contribution! I made some suggestions that I think should be addressed before merging.

I just committed the changes. Thanks for the suggestions! The translation is much better now.

**Comment by carburo** [comment]
Looks great! Thanks.
Merging

**Pull request merged**